pom.xml

```diff
@@ -50,13 +50,19 @@
             <artifactId>configuration-as-code</artifactId>
             <version>${configuration-as-code.version}</version>
             <scope>test</scope>
-          </dependency>
-          <dependency>
+        </dependency>
+        <dependency>
             <groupId>io.jenkins.configuration-as-code</groupId>
             <artifactId>test-harness</artifactId>
             <version>${configuration-as-code.version}</version>
             <scope>test</scope>
-          </dependency>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>${mockito.version}</version>
```

Comment:
Version is optional. Just remove the tag.

Example on other plugin : https://github.com/jenkinsci/openshift-k8s-credentials-plugin/blob/main/pom.xml#L65